google/cloud/pubsub/samples/samples.cc

```diff
@@ -2123,6 +2124,8 @@ void AutoRunAvro(
 
   std::cout << "\nRunning DeleteSchema() sample [avro]" << std::endl;
   DeleteSchema(schema_admin, {project_id, avro_schema_id});
+
+  CleanupSchemas(schema_admin, project_id, absl::Now());
```

Comment:
Please move this to the beginning of the function.  We want this to run first so we can recover quota space, if we ever run out, before starting the test.